hs_labels/views/__init__.py

```diff
@@ -6,9 +6,10 @@
 from hs_core.views import authorize
 
 
-def resource_labeling_action(request, shortkey, *args, **kwargs):
+def resource_labeling_action(request, shortkey=None, *args, **kwargs):
     """
     This must be a POST request. The following data needs to be passed as part of the POST request:
+	shortkey: Id of the resource (Must be None when the label_type is 'SAVEDLABEL')
     action: resource labeling action - has to be either 'CREATE' or 'DELETE'
     label_type: type of label - one of these: 'LABEL', 'FAVORITE', 'MINE', 'FOLDER', 'SAVEDLABEL'
```

Comment:
Comment: I need to check carefully whether the folder abstraction is being used. I thought it was unused.